src/edu/csus/ecs/pc2/ui/TestResultsFrame.java

```diff
@@ -102,5 +102,18 @@ public void setValidatorOutputFileNames(String[] filenames) {
         getMultiTestSetOutputViewerPane().setValidatorOutputFileNames(filenames);
     }
 
+    /**
+     * Enables/Disables Execute All button
+    */
+    public void setExecuteAllButton(boolean b) {
+        getMultiTestSetOutputViewerPane().setExecuteAllButton(b);
+    }
+
+    /**
+     * Adds action listeners to Execute All Button
+    */
+    public void addActionListenerToExecuteAll(java.awt.event.ActionListener e) {
```

Comment:
As the above comment says, this method is adding something (a listener) to a BUTTON.  However, the method name doesn't make that clear;  "ExecuteAll" could refer to some other type of object.  I would recommend changing the name of this method (and the one it calls in the following line) to `addActionListenerToExecuteAllButton()`.

Reply:
Updated